app/controllers/storage_tables/disk_controller.rb

```diff
@@ -5,8 +5,10 @@ module StorageTables
   # This means using expiring, signed URLs that are meant for immediate access, not permanent linking.
   # Always go through the BlobsController, or your own authenticated controller, rather than directly
   # to the service URL.
-  class DiskController < ActiveStorage::BaseController
-    include StorageTables::SetCurrent
+  class DiskController < BaseController
```

Comment:
We verliezen nu de `show`, is dat bewust?

Reply:
Nee die verliezen we niet, die hadden we sowieso al niet. Hij switched nu alleen van base controller. De show gebruikten we niet. Die wil ik nog wel toevoegen, maar is op dit moment niet nodig